modules/katamari/src/main/ts/ephox/katamari/api/Obj.ts

```diff
@@ -37,41 +42,39 @@ export const tupleMap = <R, T>(obj: T, f: (value: T[keyof T], key: string) => {k
   return r;
 };
 
-const objAcc = <K extends number | string | symbol, V>(r: Record<K, V>) => (x: V, i: K): void => {
+const objAcc = <T extends {}>(r: T) => (x: T[keyof T], i: keyof T): void => {
   r[i] = x;
 };
 
-const internalFilter = <V>(obj: Record<string, V>, pred: (value: V, key: string) => boolean, onTrue: (value: V, key: string) => void, onFalse: (value: V, key: string) => void) => {
-  const r: Record<string, V> = {};
+const internalFilter = <T extends {}>(obj: T, pred: ObjPredicate<T>, onTrue: ObjCallback<T>, onFalse: ObjCallback<T>) => {
   each(obj, (x, i) => {
     (pred(x, i) ? onTrue : onFalse)(x, i);
   });
-  return r;
```

Comment:
Removed this as it's only ever used internally and the return value is never used... not to mention the return value was always `{}` anyways.